src/cleanup_rules/swift/rules.toml

```diff
@@ -874,6 +874,150 @@ not_contains = [
     )""",
 ]
 
+#
+# Before 
+#   true == true
+#   (true) == true
+#   true == (true)
+#   (true) == (true)
+#   false == false
+#   (false) == false
+#   false == (false)
+#   (false) == (false)
+# After 
+#   true
+#
+[[rules]]
+name = "boolean_equality_expression_same_literals"
+query = """(
+    (equality_expression
+            lhs: [(boolean_literal) @left
+                  (tuple_expression
+                    value: (boolean_literal) @left
+                  )
+                 ] 
+            rhs: [(boolean_literal) @right
+                  (tuple_expression
+                    value: (boolean_literal) @right
+                  )
+                 ] 
+    ) @cond
+    (#eq? @left @right)
+)"""
+groups = ["boolean_expression_simplify"]
+replace_node = "cond"
+replace = "true"
+is_seed_rule = false
+
+#
+# Before 
+#   true == false
+#   false == true
+#   (true) == false
+#   (false) == true
+#   true == (false)
+#   false == (true)
+#   (false) == (true)
+#   (true) == (false)
+# After 
+#   false
+#
+[[rules]]
+name = "boolean_equality_expression_opposite_literals"
```

Comment:
Here please add support for all literals supported by swift - string, number, boolean, ...